static/app/bootstrap/exportGlobals.tsx

```diff
@@ -54,7 +56,77 @@ globals.SentryApp = {
   Modal: require('app/actionCreators/modal'),
 };
 
-// Make globals available on the window object
-Object.keys(globals).forEach(name => (window[name] = globals[name]));
+/**
+ * Wrap export so that we can track usage of these globals to determine how we want to handle deprecatation.
+ * These are sent to Sentry install, which then checks to see if SENTRY_BEACON is enabled
+ * in order to make a request to the SaaS beacon.
+ */
+let _beaconComponents: {component: string; stack: string}[] = [];
+const makeBeaconRequest = throttle(
+  async () => {
+    const api = new Client();
+
+    const components = _beaconComponents;
+    _beaconComponents = [];
+    try {
+      await api.requestPromise('/api/0/internal/beacon/', {
+        method: 'POST',
+        data: {
+          // Limit to first 20 components... if there are more than 20, then something
+          // is probably wrong.
+          batch_data: components.slice(0, 20).map(component => ({
+            description: 'SentryApp',
+            ...component,
+          })),
```

Comment:
Instead of trying to send > 20 and failing, set an upper limit on # of results so that we at least have partial results (although it shouldn't hit this max in the first place).